1355406

commit 1355406
Author: Athanatos123 <alain.delree@gmail.com>
Date:   Sun Aug 23 20:30:23 2026 +0200

    Retire .assetsignore du suivi (ajouté par erreur dans le commit de backup de l'issue #233, sans rapport avec AlChess)

diff --git a/.assetsignore b/.assetsignore
deleted file mode 100644
index c9a2fd3..0000000
--- a/.assetsignore
+++ /dev/null
@@ -1 +0,0 @@
-.wrangler/tmp/**
